scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -729,6 +759,104 @@ private void removeJobRuntimeData(Session session, long jobId) {
         session.getNamedQuery("deleteSelectorData").setParameter("jobId", jobId).executeUpdate();
     }
 
+    private void deleteInconsistentData(Session session) {
```

Comment:
We could remove it, but if the scheduler is stopped in a middle of a housekeeping, don't we have a risk to leave in the DB some unwanted tables ? With this mechanism at least we know that the next housekeeping will remove junk